packages/react-components/src/localization/locales/en-US/strings.json

```
@@ -571,5 +571,14 @@
   "verticalGallery": {
     "leftNavButtonAriaLabel": "previous page",
     "rightNavButtonAriaLabel": "next page"
+  },
+  "MeetingConferencePhoneInfo": {
+    "meetingConferencePhoneInfoModalTitle": "Conference phone info",
+    "meetingConferencePhoneInfoModalDialIn": "Dial number",
+    "meetingConferencePhoneInfoModalMeetingId": "Enter meeting ID",
+    "meetingConferencePhoneInfoModalWait": "Wait to be admitted by organizer",
+    "meetingConferencePhoneInfoModalNoPhoneAvailable": "No phone available",
+    "meetingConferencePhoneInfoModalTollFree": "(Toll-free)",
```

Avoid having just part of the string in the locale files and then manually stitching together, it's hard for us to know we concatenate correctly in all languages. Instead include the whole string and so use parameters:
```suggestion
    "meetingConferencePhoneInfoModalDialInValue": "{phoneNumber} (Toll-free) {country}, {city}",
```

---

Thanks